packages/html/src/tabstrip/tests/tabstrip-closable.tsx

```diff
@@ -0,0 +1,80 @@
+import { TabStripBottom, TabStripContent, TabStripItem, TabStripLeft, TabStripNormal, TabStripRight } from '..';
+
+export default () => (
+    <>
+        <div id="test-area" className="k-d-grid k-grid-cols-2">
+            <section>
+                <TabStripNormal
```

Comment:
As this test demonstrates scrollable tabs, maybe we could use TabStripClosable instead?